modin/core/execution/dask/common/task_wrapper.py

```diff
@@ -62,3 +67,22 @@ def materialize(cls, future):
         """
         client = default_client()
         return client.gather(future)
+
+    @classmethod
+    def scatter(cls, data, **kwargs):
```

Comment:
I prefer to call this `put`. It will be a little weird to say `DaskTask.put` or `DaskTask.scatter` because tasks aren't an object store.

Reply:
What about using `DaskWrapper` class name instead of `DaskTask`?

Reply:
Will change to `put`.

Reply:
@devin-petersohn, ping